logicrunner/executionbroker_test.go

```diff
@@ -545,16 +530,14 @@ func TestExecutionBroker_ExecuteImmutable(t *testing.T) {
 	ctx := inslogger.TestContext(t)
 	mc := minimock.NewController(t)
 	defer mc.Finish()
-	defer mc.Wait(1 * time.Second)
+	defer mc.Wait(1 * time.Minute)
```

Comment:
Minute? is it ok?
QA asked to add comments for sleeps in tests

Reply:
It is ok, it's waiting for all background tasks to be finished